File: Python/armstrongNumber.py
def armstrongNumber(number):
    digits = len(str(number))
    
    sum = 0 
    for i in range(0,digits):
        strnumber = str(number)
        
        sum = int(strnumber[i]) ** digits + sum
        # print(sum)
    if sum == number:
        return f"armstron number"
    else: return f"not a armstong numbr"

    

print(armstrongNumber(9474))
        

File: Javascript/armstrongNumber.js
console.log(Math.pow(5,2))

function armstrongNumber(number){
    let stringNumber = number.toString()
    let numOfDigits = stringNumber.length
    let sum = 0 

    for (let i=0;i<numOfDigits;i++){

        sum = parseInt(stringNumber[i]) ** numOfDigits + sum 
    }
    if (sum === number){
        return `armstrong number`
    }
    else{
        return `not armstron number`
    }
}

// console.log(armstrongNumber(15322))